src/api/guardian.js

```diff
@@ -1,16 +1,24 @@
 const Guardian = require("guardian-js");
 const guardian_filter = require("./guardian_filter");
+let allArticles = [];
 
-
-const guardianListing = (query, cb) => {
- 
+const guardianListing = (query, page, cb) => {
   const welvonAPI = new Guardian(process.env.GUARDIAN_KEY, false);
   welvonAPI.content
     .search(query, {
-      "show-fields": "headline,trailText,thumbnail,bodyText,page,lastModified"
+      "show-fields": "headline,trailText,thumbnail,bodyText,page,lastModified",
+      page: page
     })
     .then(response => {
-      cb(JSON.parse(response.body.response));
+      json_res = JSON.parse(response.body).response;
```

Comment:
what is json_res ? what kind of variable ?

Reply:
the JSON reponse of guardian API

Reply:
Also why is it snake_cased when all your other variables are camelCased?